entity-service/src/test/kotlin/com/egm/stellio/entity/config/MockSecurityConfig.kt

```diff
@@ -0,0 +1,24 @@
+package com.egm.stellio.entity.config
```

Comment:
it is the same as used in subscription-service, what about putting it in the testFixtures of the shared module?